in_progress/roles.md

```diff
@@ -0,0 +1,109 @@
+
+As a whole, we are a consensus-based community project.
+However, any subproject within the scverse can have their own governance system as long as they do not conflict with the rules set out here.
+Anyone with an interest in the project can join the community, contribute to the project design, and participate in the decision making process.
+This document describes how that participation takes place, how to find consensus, and how deadlocks are resolved.
+
+# Roles And Responsibilities
+
+## The Community
+
+The scverse community consists of anyone using or working with the project in any way.
+
+## Contributors/ Project developers
+
+A community member can become a contributor by interacting directly with the project in concrete ways, such as:
+
+- proposing, discussing, or reviewing a change to the code, documentation, or specification via a GitHub pull request to any https://github.com/scverse repository;
+- reporting a GitHub issue to any https://github.com/scverse repository;
+- discussing examples or usage issues on any of our discussion platforms
+
+among other possibilities.
+Any community member can become a contributor, and all are encouraged to do so.
+By contributing to the project, community members can directly help to shape its future.
+
+All community members are required to adhere by our code of conduct (**TODO: add link**).
+Potential contributors are encouraged to read the Contributing Guide (**TODO: add link**).
+
+A community member becomes a contributor when the following criteria are met:
+
+- At least two core developers from separate employers agree membership is a good idea.
+- The new member has supported the project several times, either through code or otherwise
+
+## Core team
+
+The core team are community members that have demonstrated continued commitment to the project through ongoing contributions.
+They have shown they can be trusted to maintain the scverse with care.
+Team members appear as organization members on the scverse [GitHub organization](https://github.com/orgs/scverse/people) and are on our [@scverse/core](https://github.com/orgs/scverse/teams/core) GitHub team (**TODO**).
+
+There are sub-roles within the core team depending on the kinds covering different kinds of contributions and responsibilities.
+
+### Core developers
+
+Core developers are members of the core team who have made signifigant code contributions to the `scverse`.
+Becoming a core developer allows contributors to merge approved pull requests, cast votes for and against merging a pull-request, and be involved in deciding major changes to the API, and thereby more easily carry on with their project related activities. 
+Core developers are on our [@scverse/core-devs](https://github.com/orgs/scverse/teams/core-devs) GitHub team (**TODO**).
+Core developers are expected to review code contributions while adhering to the [core developer guide](CORE_DEV_GUIDE.md) (**TODO**).
```

Comment:
I think you'd have areas of expertise which we might want input on. Or a particular feature you'd like to get merged.

One case here, once Daniella is back from vacation, scvi-tools core devs can get the scIB tools metrics merged faster by helping with the review. At least, that's what I'd be willing to do with scanpy – I'd be open to some level of per-project rules on "who can merge".

Reply:
If there's a particular feature I want merged in scanpy and then I go solo and write and commit it that seems bad. This language makes it seem I can approve my own PR in e.g., Squidpy.

Reply:
> If there's a particular feature I want merged in scanpy and then I go solo and write and commit it that seems bad

I think I would agree this would be bad. The idea is more that since you are one of the trusted core devs we assume you wouldn't abuse this ability.

It lets us say "hey can you review this, and merge when it's done", or lets you fix a critical bug while someone else is on vacation.